torch/_inductor/fx_passes/joint_graph.py

```diff
@@ -620,7 +620,7 @@ def repl(inp, other):
             inp = inp.to(dtype)
 
         sign: Union[int, float, torch.Tensor]
-        if isinstance(other, (int, float)):
+        if isinstance(other, (int, float, torch.SymInt, torch.SymFloat)):
```

Comment:
should we also update `mul_softmax_pattern` ?

Reply:
Yes, updated just now.